diff --git a/src/SerialTest.pro b/src/SerialTest.pro
index a2c671b..a7ea9a8 100644
--- a/src/SerialTest.pro
+++ b/src/SerialTest.pro
@@ -77,7 +77,7 @@ RC_ICONS = icon/icon.ico
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
 
 # Remember to change version in AndroidManifest.xml
@@ -137,5 +137,5 @@ exists(qcustomplot.cpp) {
         else: QCPLIB = qcustomplot
     }
 
-    LIBS += -L./ -l$$QCPLIB
+    LIBS += -L/app/lib/ -L./ -l$$QCPLIB
 }
